### ISSUE TITLE
Datadog::SyncWriter is broken

### ISSUE THREAD

**Issue by Yurokle** [DataDog/dd-trace-rb#899]
I'm seeing following error when manually setting SyncWriter:
```
An error occurred while loading spec_helper.
Failure/Error: if writer.priority_sampler.nil?

NoMethodError:
  undefined method `priority_sampler' for #<Datadog::SyncWriter:0x00007fa7012a9c48>
# /Users/ynaidyon/.rvm/gems/ruby-2.6.5/gems/ddtrace-0.30.0/lib/ddtrace/tracer.rb:410:in `configure_writer'
```

The problem seems to be this line: https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/tracer.rb#L367 since there is no`SyncWriter#priority_sampler` method.

**Comment by delner** [comment]
Thanks for the report @Yurokle! I'll get started on a fix for this and let you know as soon as I have it ready.

**Comment by delner** [comment]
Opened #900 which should fix this.

**Comment by delner** [comment]
This was released with 0.30.1. Let me know if it works for you!
